src/GoodsList.tsx

```diff
@@ -8,7 +8,13 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
+        key={good.id}
+        data-cy="good"
+        style={
+          { color: good.color }
```

Comment:
move this to scss
you don't usually want to style anything outside of scss files

Reply:
do you understand why?

Reply:
@mykola-koval , it`s compiled to inline style